### PR TITLE
V3.0 - CSS changes

### PR DESCRIPTION
**Pull request by kokers**
Leveraged built in WP UI, reorganised CSS vol 1, autoprefixer

### CODE REVIEW

**Comment by kokers** [comment]
@hailite we need to enqueue style litespeed-legacy.css for WP version < 4.2

File: litespeed-cache/var/crawlermap.data.meta [hunk 1]
```diff
@@ -0,0 +1 @@
+{"list_size":7,"last_update_time":1565545079,"curr_crawler":0,"curr_crawler_beginning_time":1565545078,"last_pos":0,"last_count":1,"last_crawled":7,"last_start_time":1565545078,"last_status":"stopped","is_running":0,"end_reason":"Crawler #1 reached end of sitemap file.","meta_save_time":1565545079,"pos_reset_check":1565545083,"done":"touchedEnd","this_full_beginning_time":1565545078,"last_full_time_cost":1,"last_crawler_total_cost":1,"file_time":1565545078}
```
Comment (lucasRolff) [inline litespeed-cache/var/crawlermap.data.meta]: Probably don't wanna have this in your PR

File: litespeed-cache/var/crawlermap.data.meta.reset [hunk 2]
```diff
@@ -0,0 +1 @@
+1565545600
```
Comment (lucasRolff) [inline litespeed-cache/var/crawlermap.data.meta.reset]: Probably don't wanna have this in your PR

File: litespeed-cache/var/crawlermap.data.blacklist [hunk 3]
```diff
@@ -0,0 +1 @@
+/checkout/
```
Comment (lucasRolff) [inline litespeed-cache/var/crawlermap.data.blacklist]: Probably don't wanna have this in your PR

File: litespeed-cache/var/crawlermap.data [hunk 4]
```diff
@@ -0,0 +1,7 @@
+/shop/
```
Comment (lucasRolff) [inline litespeed-cache/var/crawlermap.data]: Probably don't wanna have this in your PR

File: litespeed-cache/var/crawlermap.data.meta (hunk 1 above)
Comment (kokers) [reply litespeed-cache/var/crawlermap.data.meta]: Good call :o) thanks, need to add it to gitignore

**Pull request merged**